app/adapters/rating.js

```diff
@@ -0,0 +1,6 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  // host: 'http://skill-inventory.herokuapp.com',
```

Comment:
I am pretty sure you can do this conditionally by looking at the current environment—a lot like you would do it in Rails.